gpu-jobs.md

```diff
@@ -55,8 +67,45 @@ capabilities that are available to any CHTC user (as of 2/7/2020):
     <td>yes</td>
   </tr>
   <tr>
-    <td>4</td>
```

Comment:
The `gpu2000, gpu2001` servers are technically not owned by the GPU Lab but will be managed under GPU Lab policies. Making that distinction could be confusing to users.  Perhaps we should note it as an HTML comment.

Reply:
I totally made up which server went where. 😂 I can change this.